changelog.d/12037.bugfix

```diff
@@ -0,0 +1 @@
+Clean up after a long-standing bug that was fixed by accident in Synapse 1.44.0 where integers could be inserted into the `event_search` table when using sqlite.
```

Comment:
The wording's a little clumsy here. Suggestions are welcome!

Reply:
Maybe this is a little cleaner? Just a thought!

```suggestion
Clean up after a long-standing bug where integers could be inserted into the `event_search` table when using sqlite. This bug was fixed by accident in Synapse 1.44.0
```

Reply:
Would it help to mention symptoms?

> Fixed a long-standing bug where wrong data could be inserted in the `event_search` table when using sqlite. This could block running `synapse_port_db` with an "argument of type 'int' is not iterable" error.

Reply:
Thank you both for the suggestions!